### ISSUE TITLE
Error sending back list URL 

### ISSUE THREAD

**Issue by mikegyi** [revolunet/react-mailchimp-subscribe#11]
Mailchimp 'already subscribed' error sends back code in the string 

<img width="1617" alt="screen shot 2018-04-09 at 22 42 46" src="https://user-images.githubusercontent.com/8915359/38524827-e28e2bdc-3c47-11e8-8fa4-0f36cb6dc03b.png">

**Comment by revolunet** [comment]
indeed.... we should maybe use `dangerouslySetInnerHTML` here : https://github.com/revolunet/react-mailchimp-subscribe/blob/master/src/SimpleForm.js#L15-L17
